docs/notebooks/basic-scrna-tutorial.ipynb

```diff
@@ -0,0 +1,1193 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # CPM count depth scaling
</pre><p>Comment not correct any more, this is proportional fitting.</p><p>Consider explicit <code>target_sum=None</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>